src/screens/AirdropThankYou/AirdropThankYouScreen.tsx

```diff
@@ -46,19 +46,17 @@ class AirdropThankYouScreen extends Component<Props> {
         header={<Header isBackArrow title={i18n.airdrop.title} />}
         footer={
           <View style={styles.buttonsContainer}>
-            <Button
-              testID="cancel-button"
-              type="outline"
-              style={styles.iDontCareButton}
-              onPress={this.navigateBack}
-              title={i18n.airdrop.itIsAlive.iDontCare}
-            />
-            <Button
-              testID="learn-more"
-              onPress={this.goToLearnMore}
-              style={styles.learnMoreButton}
-              title={i18n.airdrop.itIsAlive.learnMore}
-            />
+            <View style={styles.iDontCareButtonContainer}>
```

Comment:
Just wondering if it's a good idea for sure to name containers or buttons based on the copy. Today it's "I don't care button" but tomorrow it might change

Reply:
I see your point, but I also don't think it won't change and if it will, changing it in loc file will take like 10 seconds so lets just keep it